docs/en_US/reference.rst

```diff
@@ -10,3 +10,4 @@ References
     NNI Annotation <Tutorial/AnnotationSpec>
     SDK API References <sdk_reference>
     Supported Framework Library <SupportedFramework_Library>
+    RESTful API <RestAPI>
```

Comment:
Oh I found REST API is not in the same topic of other pages.
SDK API is designed for _users_, while REST API is meant to be used by _developers.
I suggest to put the doc into a wiki page.